src/python/pants/backend/scala/subsystems/scalac.py

```diff
@@ -23,6 +23,18 @@ class Scalac(Subsystem):
 
     args = ArgsListOption(example="-encoding UTF-8")
```

Comment:
What are your thoughts on deprecating `--scala-args`? If so, this option will need to be marked with the `removal_version` and `removal_hint` attributes.